src/selectors/app.js

```diff
@@ -52,6 +56,210 @@ export const getTrackThreadHeights: Selector<
 export const getIsNewlyPublished: Selector<boolean> = state =>
   getApp(state).isNewlyPublished;
 
+export const getComputedHiddenGlobalTracks: Selector<
```

Comment:
TODO - I'd like to test this more. Please don't resolve this comment until I get a chance to. I don't know if I'll have time today.